src/main/java/com/abdullah/coding/challenge/util/TimeUtil.java

```diff
@@ -0,0 +1,32 @@
+package com.abdullah.coding.challenge.util;
+
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+
+public class TimeUtil {
+
+    private TimeUtil() {
+    }
+
+    public static boolean isValidTimeStamp(String time) {
+        return time.length() == 16;
+    }
+    public static Timestamp getTimeStamp(String time) {
```

Comment:
1. Explain this?

Reply:
It only converts the received date from client to the format that application and DB accepts. Again, this is just a quick solution, It would be more appropriate to use a DateTime library here.